ssh/tailssh/tailssh_test.go

```diff
@@ -392,21 +393,24 @@ func TestSSHAuthFlow(t *testing.T) {
 					"check2": acceptRule.Action,
 				},
 			},
-			wantBanner: "url-here",
+			// TODO(maisem): make this test pass.
```

Comment:
Does it pass? Delete Todo before merging.

Reply:
Hah, I added this todo when the test wasn't working. It works now.